hw/xbox/mcpx_apu.c

```diff
@@ -24,6 +24,8 @@
 #include "cpu.h"
 #include "hw/xbox/dsp/dsp.h"
 
```

Comment:
Nit: newline

Reply:
Same style as https://github.com/xqemu/xqemu/blob/master/hw/xbox/nv2a/nv2a.c#L41

Reply:
I prefer the includes to not be double spaced like this so I'd remove the newline in nv2a.c as well. But I'm not too concerned about it, just a nit.